src/js/jsx/Main.jsx

```diff
@@ -145,8 +145,8 @@ define(function (require, exports, module) {
 
         render: function () {
             var hasDocument = this.getFlux().store("application").getCurrentDocument();
-            
-            var className = classnames({
+
+            var className = classnames(window.navigator.language, {
```

Comment:
You need to chop this string down so that it works in all German-language or French-language or Japanese-language locales. Quoting the docs [here](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language):

    Examples of valid language codes include "en", "en-US", "fr", "es-ES", etc.

As discussed previously, I suggest copying [this sniipet](https://github.com/adobe-photoshop/spaces-design/blob/4a0bf29d75535a49becf21a2b3c6ca2ab4cb8f23/src/index-build.html#L31-L43) into a new property called "language" in `js/util/system`.